src/core/custom-elements/rs-changelog.js

```diff
@@ -0,0 +1,85 @@
+// @ts-check
+/**
+ * Fetches commits between two commitish (tag or commit) - `from` and `to=HEAD`
+ * and shows them as a list of "changes during" those commitish. If a filter
+ * function is provided by the user, it is used to filter the commits that are
+ * to be shown. Otherwise, all commits are shown.
+ *
+ * @typedef {{message: string, hash: string}} Commit
+ */
+import { github } from "../github.js";
+import { hyperHTML } from "../import-maps.js";
+import { showInlineError } from "../utils.js";
+
+export const name = "rs-changelog";
+
+export default class ChangelogElement extends HTMLElement {
+  constructor() {
+    super();
+    this.props = {
+      from: this.getAttribute("from"),
+      to: this.getAttribute("to") || "HEAD",
+      /** @type {(commit: Commit) => boolean} */
+      filter:
+        typeof window[this.getAttribute("filter")] === "function"
+          ? window[this.getAttribute("filter")]
```

Comment:
This is where I feel weirdest... Could we make this attribute as regex? I think a regex should be sufficient for this purpose.

Reply:
We decided to not go with regex as a function will be simpler (and less prone to mistakes) and more flexible (can use something more `commit.message` if needed).

Reply:
Might be a good idea to support both?

Reply:
What I feel weird is that this allows calling arbitrary global function, which is a bit scary... No proof that this can be a problem, though. Just me being scared.

Reply:
It's not arbitrary global function per se, author will mention which function will be called.

Reply:
Hmmm, given that other frameworks also do some similar thing, I would say this is okay. At least we don't use `eval()` 😂

Reply:
How about something like: :thinking: 
``` html
<rs-changelog from="tag">
  <script type="module">
    export default function filter(){}
  </script>
</rs-changelog>
```